src/NServiceBus.Core/DelayedDelivery/TimeoutManager/TimeoutManager.cs

```diff
@@ -102,6 +101,28 @@ static void SetupStorageSatellite(FeatureConfigurationContext context, Transport
             context.Settings.Get<TimeoutManagerAddressConfiguration>().Set(satelliteAddress);
         }
 
+        static void SetupLegacyTimeoutsSatellite(FeatureConfigurationContext context, TransportTransactionMode requiredTransactionMode)
```

Comment:
I'd call this `SetupLegacyRetriesSatellite` since this is not in anyway connected to the timeout manager =>which I guess means we should do this in the Recoverability feature instead?